syft/frameworks/crypten/__init__.py

```diff
@@ -1,13 +1,13 @@
 import torch
-import syft
-from syft.frameworks.crypten.context import toy_func, run_party
+import syft as sy
+from syft.frameworks.crypten.context import run_party
 import crypten.communicator as comm
 import crypten
 
-
-def load(tag: str, src: int, id_worker: int):
+def load(tag: str, src: int, id_worker: int, size: tuple):
```

Comment:
We may not know the size of loaded tensors so we shouldn't have `size` as a required parameter, instead, it should be optional, and we either exchange messages in case size is not provided, or doesn't exchange messages at all if it's not.